templates/helm/orchestration-workflows/encode/templates/encode.yaml

```diff
@@ -34,6 +35,14 @@ spec:
           - kubeSecretKey: AWS_SECRET_ACCESS_KEY
             path: {{ $path }}
             vaultKey: secret_access_key
+      {{- if $smokeTest }}
+      - secretName: smoke-test-secrets
+        nameSpace: encode
+        vals:
+          - kubeSecretKey: slack-url
+            path: secret/dsde/monster/{{ .Values.env }}/slack-notifier
```

Comment:
This is (unfortunately) manually-generated. What's the best way for me to document how to find the Slack App I created?